test/runtests.jl

```diff
@@ -12,8 +12,8 @@ function Base.isapprox(seq1::AbstractDict, seq2::AbstractDict)
     for (k1,v1) in seq1
         k1 ∈ keys(seq2) || (println("key $k1 missing from seq2"); return false)
         if v1 isa AbstractDict
-            return isapprox(v1, seq2[k1])
```

Comment:
with this change we no longer recursively check sub-dictionaries. Isn't that a problem for `arrays` and `info` too? It should probably be something like `if !isapprox(v1, seq2[k1]) return false`

Reply:
I think `≈` and `isapprox` are equivalent, so the new line is still recursively calling this function. I just changed the syntax to be more consistent with the rest of the function.